daemon/kill.go

```diff
@@ -105,6 +115,9 @@ func (daemon *Daemon) Kill(container *container.Container) error {
 		return errNotRunning{container.ID}
 	}
 
+	// FIXME(vdemeester) should we disable restartpolicy for SIGKILL or not ?
```

Comment:
@mlaventure you mean the SIGKILL sent after the grace period right ?


I'm not really sure in fact. Naively, I would consider SIGKILL as the *fatal* signal, and thus, the only signal that always disable restart policy.